semgrep/semgrep/core_runner.py

```diff
@@ -273,7 +260,7 @@ def _run_rules(
                     if equivalences:
                         cmd += ["-equivalences", equiv_fout.name]
                     cmd += ["-j", str(self._jobs)]
-                    cmd += [*self.targeting_options, *[str(path) for path in targets]]
+                    cmd += [str(path) for path in targets]
```

Comment:
this will break on large repos

Reply:
Yeahp. was thinking of using the -target-file argument i added to semgrep-core in a follow up PR